agentos/templates/MLProject

```diff
@@ -0,0 +1,10 @@
+{file_header}
```

Comment:
Is MLflow still a dependency given the current state of the CLI? Or did we get rid of the use of MLflow for running agents under certain conditions (which is how it used to be used)? ...in which case we can remove MLflow as a dependency and remove this template file too?

Reply:
Yeah, I think this and conda are both currently unused.  Removing for now although I suspect MLFlow or both will be back.